jekyll/_posts/2021-07-21-Nim-Efficienct-Expressive-Elecgant-Benchmarking.md

```diff
@@ -0,0 +1,369 @@
+| title | author | excerpt |
+| :---: | :----: | :---: |
+| Benchmarking the Beast | W. Gordon Goodsman (GordonBGood) | use CPU caches efficiently |
+
+Guest post
+This is a guest post by W. Gordon Goodsman (GordonBGood). If you would like to publish articles as a guest author on nim-lang.org then get in touch with us via Twitter or otherwise.
+
+# Nim: Efficient, Expressive, Elegant - Comparing these Aspects
+
+## Introduction
+
+This will be a series of blog posts showing what makes Nim such a great general purpose language that is true to its goals of being efficient, expressive, and elegant.  This first article of the series developes a benchmark program that demonstrates all three.  Further articles in the series will, first, move into a more ambitious application developed from the benchmark, then show that Nim has overcome many/most of the reasons programmers may not have taken to it in its earlier years, using more modern features to further advance the application, and finally look at new experimental features and examine how those features may make Nim competitive to any computer language in the wild, at least in the general sense as in not being specialized to any one niche.
```

Comment:
```suggestion
This will be a series of blog posts showing what makes Nim such a great general purpose language that is true to its goals of being efficient, expressive, and elegant.  This first article of the series develops a benchmark program that demonstrates all three.  Further articles in the series will, first, move into a more ambitious application developed from the benchmark, then show that Nim has overcome many/most of the reasons programmers may not have taken to it in its earlier years, using more modern features to further advance the application, and finally look at new experimental features and examine how those features may make Nim competitive to any computer language in the wild, at least in the general sense as in not being specialized to any one niche.
```

Reply:
I see the spelling correction.